packages/@okta/vuepress-site/docs/guides/embedded-siw/main/index.md

```diff
@@ -193,12 +197,17 @@ Create an app integration in the Okta org that represents the application you wa
         <button id="logout" class="button" onclick="logout()" style="display: none">Logout</button>
       </div>
       <script type="text/javascript">
-        const oktaSignIn = new OktaSignIn({
-          issuer: "https://${yourOktaDomain}/oauth2/default",
-          redirectUri: '${https://${yourAppRedirectUri} configured in your Okta OIDC app integration}',
-          clientId: "${yourClientId}",
-          useInteractionCodeFlow: true
-        });
+         const oktaSignIn = new OktaSignIn({
+            issuer: "https://${yourOktaDomain}/oauth2/default",
+            redirectUri: '${https://${yourAppRedirectUri} configured in your Okta OIDC app integration}',
+            clientId: "${yourClientId}",
+            useInteractionCodeFlow: true
+         });
+
+         // Search for URL Parameters to see if a user is being routed to the application to recover password
+         var searchParams = new URL(window.location.href).searchParams;
```

Comment:
Would it make sense to add the OTP and State to the initialization instead (where the other properties are being set)? I'll leave it up to you.

Reply:
@robertnicolo-okta I chose not to add them into the constructor because I couldn't consistently do that across all the instances where changes would need to be made. There are some pages where a config object is created by an SDK and the OktaSignIn object is constructed with the config object. In those cases, the otp and state would have to be added into the widget after construction, so I chose to do that for all examples to be consistent and to make examples easy to find. I hope that's OK?